apps/andi/lib/andi_web/controllers/auth_controller.ex

```diff
@@ -16,8 +16,11 @@ defmodule AndiWeb.AuthController do
   end
 
   def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params) do
+    {:ok, user} = Andi.Schemas.User.create_or_update(auth.uid, %{email: auth.info.email})
+
     conn
     |> TokenHandler.put_session_token(auth.credentials.token)
+    |> Plug.Conn.put_session(:user_id, user.id)
```

Comment:
Would it make sense to just put the `user` on the conn instead of the id so we wouldn't have to look the user up again on mount? Or is there some concern about that

Reply:
The user object could easily get out of date. Looking it up should keep things fresh